task2/argument_parser.py

```diff
@@ -0,0 +1,17 @@
+import argparse
+
+
+def arg_pars():
+    parser = argparse.ArgumentParser(description="Loading arguments.")
+    parser.add_argument('--debug', action='store_true', help='print debug message')
+    parser.add_argument('--ssl', action='store_true', help='Set protocol for URL.')
+    parser.add_argument('-ho', '--host', dest='host', metavar='HOST', help='Set host for URL.', required=True)
+    parser.add_argument('-po', '--port', dest='port', metavar='PORT', help='Set port for URL.', required=True)
+    parser.add_argument('-nm', '--name', dest='log_name', metavar='LOG_NAME',
```

Comment:
@patrikprovaznik this is not needed at all.. log_name should be based of the module that is being logged